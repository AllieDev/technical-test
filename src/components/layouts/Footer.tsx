import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";

export default function Footer() {
  return (
    <Box
      sx={{
        height: 100,
        backgroundColor: "primary.dark",
        display: "grid",
        placeItems: "center",
      }}
    >
      <Typography variant="body1" component="span" sx={{ color: "#fff" }}>
        Â© COPYRIGHT ALI ASGHAR SULTANI - 2023
      </Typography>
    </Box>
  );
}
