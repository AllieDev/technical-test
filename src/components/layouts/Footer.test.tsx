import Footer from "./Footer";
import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";

describe("Footer", () => {
  it("should render", () => {
    const { container } = render(<Footer />);
    expect(container.textContent).toMatchSnapshot();
  });
});
