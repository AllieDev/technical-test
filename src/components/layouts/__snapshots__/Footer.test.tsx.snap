// Vitest Snapshot v1

exports[`Footer > should render 1`] = `"© COPYRIGHT ALI ASGHAR SULTANI - 2023"`;

exports[`Loading > should render 1`] = `"© COPYRIGHT ALI ASGHAR SULTANI - 2023"`;
